src/elfpy/strategies/simple_LP.py

```diff
@@ -0,0 +1,29 @@
+from elfpy.strategies.basic import BasicPolicy
+
+
+class Policy(BasicPolicy):
```

Comment:
Is this agent type necessary? What purpose does it serve that `init_LP` and `LP_and_withdraw` does not?

Reply:
necessary for tests